composite/proof-of-reserves/src/balance.ts

```diff
@@ -45,6 +47,12 @@ export const getImpl = (options: BitcoinIndexerOptions): Execute => {
         const config = blockchair.getConfig(prefix)
         return blockchair.execute(data, config)
       }
+
+    case BitcoinIndexer.BTCCom:
```

Comment:
We now have 5 adapters here, all the same, and this switch/case is slowly outgrowing itself...

Let's add all adapters to a map, then find the one requested from a map.

```javascript
const adapter = ... // find it
return (data) => {
  const config = adapter.getConfig(prefix)
  return adapter.execute(data, config)
}
```